Jenkinsfile

```diff
@@ -0,0 +1,328 @@
+/* known issues */
+/* Parallel tasks, when executed, show up as "part of <job name>" rather than a more descriptive name. See http://stackoverflow.com/questions/37812588/how-can-i-override-the-part-of-app-pipeline-buildname */
```

Comment:
Remove this and turn it into a follow up issue, or linewrap it with a link to a jenkins upstream issue.